CHANGELOG.rst

```diff
@@ -284,6 +284,65 @@ Added
   `#350 <https://github.com/openego/powerd-data/issues/350>`_
 * Add CLI option to prefix DAG name in Airflow config
   `#1252 <https://github.com/openego/eGon-data/pull/1252>`_
+* Create new scenario status2019
```

Comment:
There should be a new section in the changelog file for each version. 
The latest version should be in the beginning. 
As long as there is no release, the changes can be also listed under "unreleased".

Reply:
Right. Changes done in 88369d69f65ec09ff08901865455413981036e28